src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -97,6 +97,23 @@ int BulletPhysicsManager::makeRigidObject(
   return newObjectID;
 }
 
+int BulletPhysicsManager::addObject(const int objectLibIndex,
+                                    DrawableGroup* drawables) {
+  // Do default load first (adds the SceneNode to the SceneGraph and computes
+  // the cumulativeBB_)
+  int objID = PhysicsManager::addObject(objectLibIndex, drawables);
+
+  // Then set the collision shape to the cumulativeBB_ if necessary
+  if (objID != ID_UNDEFINED) {
+    BulletRigidObject* bro =
+        static_cast<BulletRigidObject*>(existingObjects_.at(objID));
```

Comment:
Do we ever expect to have multiple types of rigid objects in `existingObjects_`?  (i.e. some objects are bullet objects, others are simple kinematic ones?).  If so, a dynamic cast would be safer here.

Reply:
Current design is to set the MotionType of the object to specify kinematic, dynamic or static. To enable collisions between all types, they must still be BulletRigidObject type, so I think this is fine. If the user has enabled the BulletPhysicsManager then all objects in this list are BulletRigidObject. Will likely setup a different PhysicsManager type for dealing with multi-bodies.